snackengage-core/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.github.dcendents.android-maven'
 
 android {
     compileSdkVersion 28
-    buildToolsVersion "29.0.1"
+    buildToolsVersion "29.0.2"
```

Comment:
I think this can be completely removed now

Reply:
A matter of taste. I kind of like to have control about the tooling. But feel free to remove it from the project setup.